modules/installation-about-custom-azure-vnet.adoc

```diff
@@ -0,0 +1,68 @@
+
+By deploying {product-title} into an existing Azure VNet, you might be able to avoid limit constraints in new accounts or more easily abide by the operational constraints that your company's guidelines set. This is a good option to use if you cannot obtain the infrastructure creation permissions that are required to create the VNet yourself.
+
+[IMPORTANT]
+====
+Use of an existing VNet requires the use of the updated Azure Private DNS (preview) feature. See link:https://azure.microsoft.com/en-us/updates/announcing-preview-refresh-for-azure-dns-private-zones-2/[Announcing Preview Refresh for Azure DNS Private Zones] for more information about the limits of this feature.
+====
+
+[id="installation-about-custom-azure-vnet-requirements_{context}"]
+== Requirements for using your VNet
+
+The installation program will no longer create the following components:
+* Subnets
+* Route tables
+* VNets
+* Network Security Groups
+
+If you use a custom VNet, you must correctly configure it and its subnets for the installation program and the cluster to use. The installation program cannot subdivide network ranges for the cluster to use, set route tables for the subnets, or set VNet options like DHCP, so you must do so before you install the cluster.
+
+The cluster must be able to access the resource group that contains the existing VNet and subnets. While all of the resources that the cluster creates are placed in the resource group that you specify, some network resources are used from a separate group. Some cluster Operators must be able to access resources in both resource groups. For example, the Machine API controller attaches NICS for the virtual machines that it creates to subnets from the networking resource group.
+
+Your VNet must meet the following characteristics:
+
+* The VNet’s CIDR block must contain the `Networking.MachineCIDR` range, which is the IP address pool for cluster machines.
+* The VNet and its subnets must belong to the same resource group, and the subnets must be configured to use Azure-assigned DHCP IP addresses instead of static IP addresses.
+
+You need to provide two subnets within your VNet, one for the control plane machines and one for the compute machines. Because Azure distributes machines in different availability zones within the region that you specify, your cluster will have high availability by default.
+
+To ensure that the subnets that you provide to the installation program are suitable, it confirms the following data:
+
+* All the subnets that you specify exist.
+* You provide private subnets.
+* The subnet CIDRs belong to the machine CIDR that you specified.
+* Unless you use a private cluster, you must provide exactly one public and private subnet for each availability zone. Each availability zone contains no more than one public and one private subnet.
```

Comment:
The subnets provided to the install config for Azure are both private subnets. Public IP addresses are allocated for the load balancers that sit in front of the control plane and worker nodes.